data_steward/cdr_cleaner/cleaning_rules/clean_ppi_numeric_fields_using_parameters.py

```diff
@@ -166,19 +163,16 @@ def setup_validation(self, client):
         """
         Run required steps for validation setup
         """
-        pass
+        raise NotImplementedError("Please fix me.")
 
     def validate_rule(self, client):
         """
         Validates the cleaning rule which deletes or updates the data from the tables
         """
-        pass
+        raise NotImplementedError("Please fix me.")
 
     def get_sandbox_tablenames(self):
-        sandbox_table_names = list()
-        sandbox_table_names.append(self._issue_numbers[0] +
-                                   self._affected_tables[0])
-        return sandbox_table_names
+        return [INVALID_VALUES_SANDBOX]
```

Comment:
```suggestion
        return f'{self._issue_numbers[0].lower()}_{self._affected_tables[0]}'
```

You can use this method to return the sandbox table name, like earlier. Since you have all the tickets listed in the class, it would be easier to find the original ticket. that shouldn't be a problem.